### ISSUE TITLE
uploading a custom logo does not change the logo on the login page

### ISSUE THREAD

**Issue by individual-it** [owncloud/web#11603]
## Describe the bug
even after uploading a custom logo, the original ownCloud logo is shown on the login page

## Steps to reproduce
1. as admin upload a custom logo in the admin settings
2. logout

## Expected behavior
the new logo should be shown on the login page

## Actual behavior
the original logo is shown (signin/v1/static/logo.svg)

## Setup
docker compose

[docker-compose.yml.txt](https://github.com/user-attachments/files/17040769/docker-compose.yml.txt)
[web.config.json](https://github.com/user-attachments/files/17040771/web.config.json)

## Additional context
theme.json:
```json
{"clients":{"web":{"defaults":{"logo":{"login":"themes/_branding/Logo-white.png","topbar":"themes/_branding/Logo-white.png"}}}},"common":{"logo":"themes/_branding/Logo-white.png"}}
```

**Comment by main1byte** [comment]
similar situation

**Comment by kulmann** [comment]
The idp in ocis is completely standalone / separated from the web ui. But you're right, meanwhile the theme is defined server side, so we could incorporate it in the built in idp. Technically more of a feature request. 😁 But yeah, user facing this is buggy behaviour.

**Comment by main1byte** [comment]
Lang?

**Comment by main1byte** [comment]
> > Lang?
> 
> Could you rephrase your question please? 😅
> 
> The next ocis rolling release (6.6 in 1,5 weeks) will automatically use the logo from the theme (= also logo upload) for the built in idp. Development is done, that's why this issue is closed now.

Can the idp language be specified in docker compose?
